internal/server/recon/golden/recognize_entities/result.json

```diff
@@ -0,0 +1,101 @@
+{
+  "queryItems": {
+    "tell me about Benzodiazepine, derivatives and their use in California": {
+      "items": [
+        {
+          "span": "tell"
+        },
+        {
+          "span": "me",
+          "places": [
```

Comment:
should here use "entities" instead of "places"?

Reply:
updated the proto message